content/F2015/F15-event-code-party.md

```diff
@@ -1,21 +1,21 @@
-title: Code Party 
```

Comment:
So now it says you deleted this file. It should look the same in this branch as it does in master